simplivity/resources/hosts.py

```diff
@@ -170,3 +170,8 @@ def remove(self, force=False, timeout=-1):
 
         self._client.do_post(method_url, data, timeout, http_headers)
         self.data = None
+
+    def get_virtual_controller_status(self):
```

Comment:
I think we should change the method name to "get_virtual_controller_shutdown_status"  or "get_ovc_shutdown_status" as it is returning only the shutdown status the OVC.

Reply:
Updated the method name to "get_virtual_controller_shutdown_status"